core/app/services/uffizzi_core/project_service.rb

```diff
@@ -36,14 +36,12 @@ def update_compose_secret_errors(project, secret)
       UffizziCore::ComposeFile::ErrorsService.update_compose_errors!(compose_file, new_errors, compose_file.content)
     end
 
-    def add_users_to_project!(project, current_user)
-      user_projects = []
+    def add_user_to_project(user, project, account)
+      is_account_owner = account.owner_id == user.id
+      role = is_account_owner ? UffizziCore::UserProject.role.admin : UffizziCore::UserProject.role.developer
 
-      current_user.default_account.memberships.where(role: UffizziCore::Membership.role.admin).map do |membership|
-        user_projects << { project: project, user: membership.user, role: UffizziCore::UserProject.role.admin }
-      end
```

Comment:
When we create a new project, we need to add all account admins to the project as admins, this is what this code is for